public/event-emitters/search-admin-event.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * When the user selects and administrative region in a country (polygon on the map).
+ */
+
+var _ = require('lodash');
+var P = require('pjs').P;
+
+var SearchAdminEvent = P({
+  key: 'SearchAdminEvent',
+  init: function(searched_admin_codes) {
+    if (!_.isArray(searched_admin_codes)) {
+      console.error("Bad argument to SearchAdminEvent", searched_admin_codes);
+    }
```

Comment:
I think I was a little weirded that this event type is an array, and not a single searched admin. Maybe we can discuss what is a more clear API. But for now, let's keep it the same; we can write tests to make sure argument types used are correct.